reference/pipeline/expressions/index.md

```diff
@@ -186,12 +186,36 @@ information from Jenkins properties files or other similar endpoints.
 
 ### #stage(String)
 
-A shortcut to get the stage by name. For example, `${#stage("Bake")}` allows
+A shortcut to get the stage by name. For example, `${#stage('Bake')}` allows
 you to access your Bake stage. Note that `#stage` is case sensitive: if your
 stage is actually named "bake", `${#stage("Bake")}` will not find it. Remember
 that the values for the stage are still under the _context_ map, so you can
 access a property via `${#stage("Bake")["context"]["desiredProperty"]}`.
 
+### #stageByRefId(String)
+
+A shortcut to get the stage by its `refId`. For example, `${#stage('3')}` allows
+you to access the stage with `refId = 3`.
+
+### #currentStage()
+
+Returns the current stage.
+
+### #stageExists(String)
+
+Checks if a given stage exists. You can search by `name` or `id`.
+Returns `true` if at least one stage is found with the `name` or `id` given.  
+Since the `id` is generated at runtime, most of the time it will make sense to search by `name` instead.
+Note that stage names are set by default so if you create a Webhook stage it will be called Webhook; 
+give the stage a unique name to be able to find it accurately.
+
```

Comment:
Suggest: "...giving the stage a unique name when you create it make it easier to find when using this helper function."

Reply:
nice, thanks!

Reply:
ignoring my typo, of course :wink: 
*"makes"